src/librpmem/rpmem_fip.c

```diff
@@ -1049,6 +1063,101 @@ rpmem_fip_persist_saw(struct rpmem_fip *fip, size_t offset,
 	return 0;
 }
 
+/*
+ * rpmem_fip_persist_inline -- (internal) perform persist operation using
+ * RDMA SEND operation with data inlined in the message buffer.
+ */
+static int
+rpmem_fip_persist_inline(struct rpmem_fip *fip, size_t offset,
+	size_t len, unsigned lane, unsigned flags)
+{
+	RPMEM_ASSERT(len <= fip->buff_size);
```

Comment:
Why to do assert?
I think the rpmem persist shall send the data in multiple chunks if the area to be persisted is bigger than the buffer in for SEND message